src/IssueViz.Security/OpenInIDE/Contract/IStatusResponse.cs

```diff
@@ -20,27 +20,10 @@
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Contract
 {
-    // TODO:
```

Comment:
I changed the names to match those used in SLI.
The icon information will be returned separately. I created another ticket for that.